diff --git a/test/test_utils.py b/test/test_utils.py
index 8e04d43..62bc4f0 100644
--- a/test/test_utils.py
+++ b/test/test_utils.py
@@ -490,20 +490,3 @@ class TestTemporaryEnvironment(BaseTest):
 
         os.unlink(tmp_path)
 
-
-class TestRpmHelper(BaseTest):
-    """ RpmHelper class tests. """
-
-    def test_is_package_installed_existing(self):
-        assert RpmHelper.is_package_installed('glibc') is True
-        assert RpmHelper.is_package_installed('coreutils') is True
-
-    def test_is_package_installed_non_existing(self):
-        assert RpmHelper.is_package_installed('non-existing-package') is False
-        assert RpmHelper.is_package_installed('another-non-existing-package') is False
-
-    def test_all_packages_installed_existing(self):
-        assert RpmHelper.all_packages_installed(['glibc', 'coreutils']) is True
-
-    def test_all_packages_installed_one_non_existing(self):
-        assert RpmHelper.all_packages_installed(['glibc', 'coreutils', 'non-existing-package']) is False
